### PR TITLE
fix exception when trying to join game with invalid id.

### PR DESCRIPTION
**Pull request by Pacane**
fix client side error when trying to join `/games/anythingAsId`

### CODE REVIEW

File: client/lib/current_game.dart [hunk 1]
```diff
@@ -8,7 +8,7 @@ class CurrentGame {
   int _gameId;
 
   void setGameId(String newGameId) {
-    _gameId = int.parse(newGameId);
+    _gameId = int.parse(newGameId, onError: (_) => null);
```
Comment (christiangoudreau) [inline client/lib/current_game.dart]: No log of the error?

Comment (meriouma) [inline client/lib/current_game.dart]: Nice, is this syntax specific to the int.parse method?

**Comment by meriouma** [comment]
LGTM

File: client/lib/current_game.dart (hunk 1 above)
Comment (Pacane) [inline client/lib/current_game.dart]: @meriouma yes, it takes an optional function called `onError`. There are several other methods who use the same pattern.

Comment (Pacane) [inline client/lib/current_game.dart]: @christiangoudreau I don't know.. Would you log it in GWTP if someone tried to access an invalid url? If someone tried to go to some gibberish url like `/patients/cfjkslfhjd` ?

Comment (christiangoudreau) [inline client/lib/current_game.dart]: Not client side, you're right, but If it were server side, yes.

**Pull request merged**